src/common/search/FilterPlays.jsx

```diff
@@ -52,39 +52,49 @@ const FilterPlaysModalBody = ({ filterQuery, setFilterQuery }) => {
     })) || []),
   ];
 
-  const levelOptions = [
-    defaultOption,
-    ...(levels?.map((level) => ({
-      label: level.level.name,
-      value: level.level.id,
-    })) || []),
-  ];
+    const levelOptions = [
+        defaultOption,
+        ...(levels?.map((level) => ({
+            label: level.level.name,
+            value: level.level.id,
+        })) || []),
+    ];
 
-  const tagOptions = [
-    defaultOption,
-    ...(tags?.map((tag) => ({
-      label: tag.tag,
-      value: tag.id,
-    })) || []),
-  ];
+    const tagOptions = [
+        defaultOption,
+        ...(tags?.map((tag) => ({
+            label: tag.tag,
+            value: tag.id,
+        })) || []),
+    ];
 
-  const languageOptions = [
-    defaultOption,
-    ...languages?.map((language) => ({
-      label: language === "ts" ? "TypeScript" : "JavaScript",
-      value: language,
-    })),
-  ];
+    const languageOptions = [
+        defaultOption,
+        ...languages?.map((language) => ({
+            label: language === "ts" ? "TypeScript" : "JavaScript",
+            value: language,
+        })),
+    ];
 
   const renderCreator = (value) => {
     const selectedCreator = creatorOptions.find((option) => option.value === value);
     return selectedCreator ? selectedCreator.label : "All";
   };
 
+  // condition to check if there is no applied filters
+  // If False, clicking on onClearAppliedFilters function linked to Clear All button
+  // will not be activated
+  const isFilterEmpty= filterQuery.level_id !== "" || filterQuery.tags.length !== 0 || filterQuery.owner_user_id !== "" || filterQuery.language !== "";
+
   return (
     <>
       <div className='form-group'>
         {loading && "Loading Data"}
+        {/* Clear All filters button */}
+        <div className={"modal-clear-filter"}>
+          <span onClick={ isFilterEmpty && onClearAppliedFilters} 
```

Comment:
need else statement also or more over if we ger rid of the `isFilterEmpty` check as mentioned above then no more condition is needed here.